docs/content/preview/yugabyte-voyager/migrate/live-fall-forward.md

```diff
@@ -507,86 +529,100 @@ Perform the following steps as part of the cutover process:
 
     {{< /warning >}}
 
-1. Stop [archive changes](#archive-changes-optional).
+### End migration
 
-{{< note >}}
-During `fall-forward synchronize`, yb-voyager creates a CDC stream ID on the target YugabyteDB database to fetch the new changes from the target database which is displayed as part of the `fall-forward synchronize` output. You need to manually delete the stream ID after `fall forward switchover` is completed.
-{{< /note >}}
+To end the migration, you need to clean up the export directory (export-dir), and Voyager state ( Voyager-related metadata) stored in the target YugabyteDB database and source-replica database.
+
+Run the `yb-voyager end migration` command to perform the clean up, and to back up the schema, data, migration reports, and log files by providing the backup related flags (mandatory) as follows:
+
+```sh
+# Replace the argument values with those applicable for your migration.
+yb-voyager end migration --export-dir <EXPORT_DIR> \
+        --backup-log-files <true, false, yes, no, 1, 0> \
+        --backup-data-files <true, false, yes, no, 1, 0> \
+        --backup-schema-files <true, false, yes, no, 1, 0> \
+        --save-migration-reports <true, false, yes, no, 1, 0> \
+        # Set optional argument to store a back up of any of the above arguments.
+        --backup-dir <BACKUP_DIR>
+```
+
+Note that after you end the migration, you will _not_ be able to continue further. If you want to back up the schema, data, log files, and the migration reports (`analyze-schema` report and `get data-migration-report` output) for future reference, the command provides an additional argument `--backup-dir`, using which you can pass the path of the directory where the backup content needs to be saved (based on what you choose to back up).
+
+Refer to [end migration](../../reference/end-migration/) for more details on the arguments.
 
 ## Limitations
 
 In addition to the Live migration [limitations](../live-migrate/#limitations), the following additional limitations apply to the fall-forward feature:
 
 - Fall-forward is unsupported with a YugabyteDB cluster running on [YugabyteDB Managed](../../../yugabyte-cloud).
-- [SSL Connectivity](../../reference/yb-voyager-cli/#ssl-connectivity) is unsupported for export or streaming events from YugabyteDB during `fall-forward synchronize`.
-- You need to manually disable constraints on the fall-forward database.
-- yb-voyager provides limited datatypes support with YugabyteDB CDC during `fall-forward synchronize` for datatypes such as DECIMAL, and Timestamp.
-- You need to manually delete the stream ID of YugabyteDB CDC created by Voyager during `fall-forward synchronize`.
+- [SSL Connectivity](../../reference/yb-voyager-cli/#ssl-connectivity) is unsupported for export or streaming events from YugabyteDB during `export data from target`.
+- yb-voyager provides limited datatypes support with YugabyteDB CDC during `export data from target` for datatypes such as DECIMAL, and Timestamp.
```

Comment:
Not sure what this means
- support for data types during CDC is limited, eg, Decimal and Timestamp are not supported; OR
- support for Decimal and Timestamp data types is limited during CDC; 
- or something else entirely